src/Microsoft.VisualStudio.SlowCheetah/Transformer/TransformerFactory.cs

```diff
@@ -89,7 +51,15 @@ public static bool IsJsonFile(string filePath)
         /// <returns>True is the file type is supported</returns>
         public static bool IsSupportedFile(string filepath)
         {
-            return IsJsonFile(filepath) || IsXmlFile(filepath);
+            foreach (ITransformer transformer in TransformerCatalog)
```

Comment:
This can be changed to a `return TransformerCatalog.Any(x => x.IsFileSupported(filePath));`